src/main/java/im/status/keycard/SecureChannel.java

```diff
@@ -176,6 +199,66 @@ public void mutuallyAuthenticate(APDU apdu) {
     respond(apdu, len, ISO7816.SW_NO_ERROR);
   }
 
+  /**
+   * Processes the LOAD_CERTS command. Copies the APDU buffer into `certs`.
+   * This function expects a DER signature and may only be called once.
+   * @param apdu the JCRE-owned APDU object.
+   */
+  public void loadCert(APDU apdu) {
+    byte[] apduBuffer = apdu.getBuffer();
+
+    final short caPubOffset = (short) ISO7816.OFFSET_CDATA;
```

Comment:
But...why

Reply:
I think this comment is out of date. I don't do that anymore